test/helpers/filter_helper_test.rb

```diff
@@ -182,4 +182,12 @@ class FilterHelperTest < ActionView::TestCase
     }
     assert_equal [{ contentTypeFilter: 'dataset' }], applied_filters
   end
+
+  test 'gdt_sources returns the label we want for MIT stuff' do
```

Comment:
No, it will return `nil`, which is why the views check that the current filter category is `sourceFilter`. However, we could do this:

```
def gdt_sources(category)
  return if category != :sourceFilter
  etc
end
```

Reply:
Yeah I think something like that would be great and allow us to slow roll out UI support for new sources if they come up in the future and we somehow didn't expect them (this seems extremely unlikely fwiw so feel free to push back on this need)

Reply:
Actually, at that point, I'd probably just switch from the hash returning the appropriate string based on the value. But the point is, if you'd rather see that logic in the helper than the views (and thus tested), we can make that happen.

Reply:
I'll see how this feels with fresh coffee in the morning. Thanks for the confirmation how the code currently works. I'll let you know if I think we should tweak the expectations tomorrow.

Reply:
@JPrevost Would you like me to refactor this method? I think we were leaning yes, but we left this in more of a 'maybe' state.

Reply:
If you don't mind I think it would be great to do that. So if we have a translation, use it. If not, just pass through whatever is in the data... if you'd prefer to open a backlog ticket for that I'm fine with this and could move towards approval. The chances of another source sneaking into the `geo` alias feel low enough to consider this backlog to me (maybe looking at it at a timdex-ui-as-a-whole perspective and not as a geo specific feature if we do that)